connectors/src/connectors/intercom/temporal/sync_help_center.ts

```diff
@@ -244,123 +181,6 @@ export async function _upsertCollection({
     });
   }
 
-  // Sync the Collection's articles
```

Comment:
This has been moved below on `upsertArticle`, called by the new activity `syncArticleBatchActivity`